src/osx/cocoa/dirdlg.mm

```diff
@@ -62,26 +65,40 @@
 
 WX_NSOpenPanel wxDirDialog::OSXCreatePanel() const
 {
-    NSOpenPanel *oPanel = [NSOpenPanel openPanel];
+    NSOpenPanel* oPanel = [NSOpenPanel openPanel];
```

Comment:
If possible, please avoid whitespace-only changes.

Reply:
In this case, I was making a lot of changes in the file and I had originally made a change here that I ended up not needing. The whitespace change was a byproduct of that. I don't think it needs feverting in the review since that will cause more noise.